src/server/views/privacy-statement.njk

```diff
@@ -0,0 +1,90 @@
+{% extends 'layouts/default.layout.njk' %}
+
+{% set pageTitle = t.site_title %}
+{% set pageBreadcrumbItems = [
+  {
+    text: t.breadcrumbs.home,
+    url: '/'
+  }, {
+    text: t.breadcrumbs.privacy_statement
+  }
+] %}
+
+{% block content %}
+    {% call components.gridRow() %}
+        {% call components.columnFull() %}
```

Comment:
Should this not be `twoThirds()`?

I have noticed that on the Cookies, Terms and A11y statement pages that we use twoThirds layout. Not not for Privacy Statement. Is this intentional?

Reply:
changed to two thirds for consistency